smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/SwiftIntegration.kt

```diff
@@ -39,19 +41,6 @@ interface SwiftIntegration {
     val order: Byte
         get() = 0
 
-    /**
-     * Preprocess the model before code generation.
-     *
-     *
-     * This can be used to remove unsupported features, remove traits
-     * from shapes (e.g., make members optional), etc.
-     *
-     * @param model model definition.
-     * @param settings Setting used to generate.
-     * @return Returns the updated model.
-     */
-    fun preprocessModel(model: Model, settings: SwiftSettings): Model = model
```

Comment:
Q: What's the reason behind factoring out preprocessing model from `SwiftIntegration` to `SwiftCodegenPlugin`?

Reply:
`SwiftIntegration::preprocessModel` is [inherited from `SmithyIntegration`](https://github.com/smithy-lang/smithy/blob/7a5efbdfd9c6128768af18fc3dc73604d871d291/smithy-codegen-core/src/main/java/software/amazon/smithy/codegen/core/SmithyIntegration.java#L141-L155), so the method declaration here is redundant.